src/shared/hooks.ts

```diff
@@ -80,16 +81,16 @@ export const useSSRData: {
   <TData, TResp, TError extends APIError = APIError>(
     dataMethod: () => Promise<APIResponse<TResp, TError>>,
     initialData: TData | undefined,
-    transformer: (resp: TResp) => TData,
+    transformer: (resp: TResp) => TData
   ): UseSSRDataHook<TData>;
   <TData, TError extends APIError = APIError>(
     dataMethod: () => Promise<APIResponse<TData, TError>>,
     initialData: TData | undefined
   ): UseSSRDataHook<TData>;
-} = <TData, TResp, TArgs, TError>(
-  dataMethod: (args?: any) => any,
+} = <TData, TResp, TError>(
```

Comment:
VSCode informs me that `TResp` and `TError` aren't being used either, so we can get rid of those too.

Reply:
2009067